<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="SRStdMultiWidget" name="multiWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
  background: rgb(0, 0, 0)
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonDefault">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonShow">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonView">
           <property name="text">
            <string>View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_5">
           <property name="text">
            <string>Tools</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_8">
           <property name="text">
            <string>Marking</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_9">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonTools">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_11">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_12">
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_13">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_14">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_15">
           <property name="text">
            <string>Align</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_16">
           <property name="text">
            <string>Planning</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_17">
           <property name="text">
            <string>Fusion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_18">
           <property name="text">
            <string>IntraOp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SRStdMultiWidget</class>
   <extends>QWidget</extends>
   <header>SRStdMultiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
